src/map/label.cpp

```diff
@@ -190,7 +200,7 @@ const terrain_label* map_labels::set_label(const map_location& loc,
 			*this, text, creator, team_name, loc, color, visible_in_fog, visible_in_shroud, immutable, category, tooltip);
 
 		// Hide the old label.
-		if(global_label != nullptr) {
+		if (global_label != nullptr) {
```

Comment:
Remove the added space.

Reply:
Done. Check once, where the spaces shouldn't be.

Reply:
I checked and removed the spaces.